spec/intro.md

```diff
@@ -10,7 +10,7 @@
    1. [`message.abnf`](message.abnf)
 1. [Formatting](formatting.md)
 1. [Errors](errors.md)
-1. [Default Function Registry](registry.md)
+1. [Default Function Registry](functions/README.md)
```

Comment:
In some teleconferences in October/November, we discussed changing from using the term "registry" to using the term "function set". I made changes _outside_ the spec, but didn't change the spec to implement that. We should probably adopt the terminology we intend to use longer-term, e.g.

```suggestion
1. [Default Function Set](functions/README.md)
```